src/domain/application/hooks/useApplications.ts

```diff
@@ -7,16 +7,21 @@ async function getApplications() {
   return data;
 }
 
+async function getApplicationsForHanke(hankeTunnus?: string) {
+  const { data } = await api.get<{ applications: Application[] }>(
```

Comment:
I'm not sure how this will handle the backend response. It is implemented like

```
{
  "applications": []
}

```

Reply:
I tried to link swagger docs here but dev environment has not updated itself for some reason.

Reply:
nevermind. I read this wrong :D